### PR TITLE
refactoring supporters block

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/karmada-io/website/issues/52

### CODE REVIEW

**Comment by RainbowMango** [comment]
Thanks @Arhell for your hard work! Take care of yourself. Have a nice weekend!

Once it's ready for review please assign it to me or @rgrupesh.

**Comment by Arhell** [comment]
/assign @RainbowMango
/assign @rgrupesh

I moved the addition of logos to the 1st place
https://github.com/karmada-io/website/pull/53/files#diff-11851fc3516533a823c71a9f770f1f12b9e9bdcab6ca99f3990466c8524ec6c2

so that they can add a logo very simply, following the example of the code that is in the file

File: src/components/supportersList.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+                    <li key={index}>
+                        <img src={logo} alt={alt}/>
+                    </li>
+                ))
+            }
+        </ul>
+    )
+}
+
+export default SupportersList
```
Comment (RainbowMango) [inline src/components/supportersList.js:18]: Should we add a new line to the end of the file?

File: src/data/supporters.js [hunk 2]
```diff
@@ -64,18 +62,4 @@ const supporters = [
...
-                    </li>
-                ))
-            }
-        </ul>
-    )
-}
-
-export default SupportersBlock
\ No newline at end of file
+export default supportersData
```
Comment (RainbowMango) [inline src/data/supporters.js:65]: ditto

File: src/components/supportersList.js (hunk 1 above)
Comment (Arhell) [reply src/components/supportersList.js]: Done

File: src/data/supporters.js (hunk 2 above)
Comment (Arhell) [reply src/data/supporters.js]: Done

**Pull request merged**